### DISCUSSION TITLE
Image gallery for posts

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#261] [Q&A]
Is there a module and shortcode to put an image gallery inside a post?

For example, in some posts that I write about cars, I would like to put an image gallery of that car in the end of the post, with a slider if possible.

**Comment by razonyang** [comment]
There is a general shortcode called [bs/img-grid](https://bootstrap.hugomods.com/docs/img-grid/), but may not suit your needs.

**Reply by LITUATUI** [reply]
Thanks. 

I think I'll use [this](https://github.com/liwenyip/hugo-easy-gallery) module for now, even if it's no longer maintained - until I find another one.

**Comment by LITUATUI** [comment]
For someone searching for a simple gallery inside a post, [this](https://hugocodex.org/add-ons/image-gallery/) can also be a solution.
